src/main/java/hudson/plugins/report/jck/ReportChartColumn.java

```diff
@@ -50,8 +50,7 @@ public ReportChartColumn() {
         return r;
     }
 
-    public int getLatestResultFailures(Job<?, ?> job) {
-        List<BuildReport> reports = getJckReport(job);
+    public int getLatestResultFailures(final List<BuildReport> reports) {
```

Comment:
Sorry. No idea what this header change willdo. Does it build? Is it working afterwards?

Reply:
See the column.jelly. Method getJckReport is called twice with the same argument (job) and is pretty expensive since it iterates through all builds 3 times. Now I call it just once in jelly and pass the result to this method. I hope this makes sence :)